### DISCUSSION TITLE
(Feature request) Spoof EID5 automatically if PEX/DEX is detected.

### DISCUSSION THREAD

**Discussion by ghost** [aldostools/webMAN-MOD#970] [Ideas]
Requires a .pkg to be installed on webMAN for this on retail to debug.

**Comment by ghost** [comment]
Notice: it'll read as DEX on Rebug 4.84.2.

**Comment by aldostools** [comment]
This is out of the scope of webMAN MOD. I don't have plan to implement this feature.
